mantidproject/mantid
# Swap Start And End Times If Start Is Greater In Muon Analysis Fitting

**Description of work.**
Start and end are interchangeable and fitting occurs successfully anyway. Decided to keep this but instead if start is greater than end to swap them round on the interface once the fitting is done.

**To test:**
1. Open the interface (interfaces -> muon -> muon analysis)
2. Load some data
3. Go to fitting tab and add flat background 
4. Change Start Time in the properties table to be greater than End Time
5. Click fit and the fit should still work and they should be swapped on the interface

Fixes #25821 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
